en/editor/publish/publish-bytedance-mini-game.md

```diff
@@ -0,0 +1,62 @@
+# Publish to ByteDance Mini Games
+
+ByteDance mini-game is developed based on the ByteDance full products, which does not require users to download, and is a brand-new game type that can be played on tap.
+
+The game submission, review and release process of the mini-games need to comply with the requirements and standard processes of the Byte official team. For specific information, please refer to the link at the end of the article.
```

Comment:
The game submission, review and release process of the mini-games need to comply with the requirements and standard processes of the Byte official team. For specific information, please refer to the link at the end of the article.
-->
The game submission, review and release process of a mini-game needs to comply with the requirements and standard processes of the Byte official specification. For specific information, please refer to the links at the end of this document.